docs/component-status.md

```diff
@@ -21,7 +21,31 @@ When we add, make significant updates, or deprecate a component we update their
     </tr>
   </thead>
   <tbody>
-    <!-- 2.6.0 -->
+    <!-- 2.7.0 -->
+    <tr>
+      <th><a href="/base/typography#heading-classes">Heading classes</a></th>
+      <td><div class="p-label--updated">Updated</div></td>
+      <td>New heading classes with numbers (<code>p-heading-1</code>, ...) were added for consistency with other class names we use.</td>
```

Comment:
`p-heading--1` double dash

Reply:
@anthonydillon 

I decided not to do this as it seems confusing when compared with other classes with numbers like `col-1`.
Also, we don't really have `p-heading` pattern, so the numbers are not really modifiers, they are integral part of the pattern name. It doesn't exist without it.

But I'm open to arguments.

Reply:
I agree the grid is a special case but we shouldn't continue to spread special cases. The reason the grid uses none BEM names (`row`, `*-col`) is due to the fact it used to come from a separate package called [Shelves](http://shelvesgrid.org/). I think the heading pattern denoted by the prefixed `p-` should continue to be a pattern or become a utility, in any case, should continue to follow the BEM naming convention.

Reply:
OK. You are right that it seems like our use of `p-heading--` classes is more like an utility than pattern. But let's not do that now.

I'll update the names to use double dashes.